test/system/250-systemd.bats

```diff
@@ -281,4 +281,30 @@ LISTEN_FDNAMES=listen_fdnames" | sort)
     is "$output" "" "output should be empty"
 }
 
+# https://github.com/containers/podman/issues/13153
+@test "podman rootless-netns slirp4netns process should be in different cgroup" {
```

Comment:
Is there any reason not to `if root; then skip "only meaningful when rootless"`?

And, is this a valid test in netavark-world? Should it be skipped if netavark? (If not, should the test be renamed?)

Reply:
Yes we can skip for rootful. It is also required for netavark, why should the test be renamed it does not mention cni?

Reply:
Probably my mistake: I don't know what slirp or cni really mean, so I assumed they were the same thing. If I understand your comment correctly, netavark replaces cni but still uses slirp. If so, then please ignore my comment.

Reply:
Yes slirp4netns is always needed for rootless networking